Week5/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/model/MovieApplication.kt

```diff
@@ -0,0 +1,20 @@
+package com.raywenderlich.myfavoritemovies.model
+
+import android.app.Application
+import android.content.Context
+import androidx.room.Room
+
+class MovieApplication : Application() {
```

Comment:
Okay so you do have this, but you still create Prefs every single time in code. It'd be better to just have prefs created here.

And the database is a lateinit var, meaning anyone can change it. It should be a `lazy` property instead.